packages/web/src/js/FileUploader.ts

```diff
@@ -212,6 +214,13 @@ class FileUploader extends BaseComponent {
     return attachmentValidationTextElement;
   }
 
+  static findValidationTextInElement(validationText: string, element: HTMLElement) {
+    const listItems = element.getElementsByTagName('li');
+    const matchingItems = Array.from(listItems).filter((item) => item.textContent?.includes(validationText));
+
+    return matchingItems.length > 0;
```

Comment:
```suggestion
    return Boolean(matchingItems.length); // or !!matchingItems.length
```

Maybe better solution because the zero is a "falsy" value. But I am leaving this up to you.